pyccel/ast/variable.py

```diff
@@ -85,6 +85,8 @@ class base if variable is an object or an object member [Default value: None]
 
     is_const: bool
         if object is a const argument of a function [Default value: False]
+    is_temp: bool
+        Indicates if the variable is created by Pyccel and it is not necessary to print [Default value: False]
```

Comment:
See my previous comment.

Further, please add empty line before `is_temp: bool` to match formatting.